pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -160,7 +160,26 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 
 	tests, err := testsForSuite()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed reading origin test suites: %w", err)
+	}
+
+	if len(os.Getenv("OPENSHIFT_SKIP_EXTERNAL_TESTS")) == 0 {
```

Comment:
Will add.

Reply:
On a second check, I don't think that matters. Since here we check if it's not set, in which case we do try to get the external binary. There's no requirement to the value of the env var, only it's presence indicates we should rely only on the built-in tests.